fastlane/lanes/release.rb

```diff
@@ -262,18 +332,40 @@ def trigger_buildkite_release_build(branch:, beta:)
   buildkite_annotate(style: 'info', context: 'trigger-release-build', message: message)
 end
 
-def create_release_backmerge_pr(version_to_merge:, next_version:)
+def create_backmerge_pr!
+  pr_urls = create_backmerge_prs
+
+  return pr_urls unless pr_urls > 1
```

Comment:
I haven't tested, but don't you need to compare the size?
```suggestion
  return pr_urls unless pr_urls.size > 1
```

Reply:
Woops! 😳 

Thanks @iangmaia

Reply:
For the record: I updated @iangmaia original suggestion from `size` to `length` and applied it from GitHub to maintain attribution.

Rationale: [`size` is an alias for `length`](https://ruby-doc.org/3.3.5/Array.html#method-i-size) and as far as I've seen, `length` is the most commonly used.